Makefile

```diff
@@ -109,6 +109,13 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+.PHONY: mod.check
+mod.check:#check go module dependencies
+	go mod verify
+	go mod tidy
```

Comment:
This check should error out if there the check fails (i mean if there is a change in go.mod/go.sum/vendor) and can you please also check if we need to run it in  API package as we also have go.mod and go.sum over there as well.

Reply:
@Madhu-1  wrote:
> This check should error out if there the check fails (i mean if there is a change in go.mod/go.sum/vendor)

I don't quite get it yet: Why should there be a change there?
Do you mean that the operations like `go mod tidy` could lead to a change there and that should result in test failure?

>  and can you please also check if we need to run it in API package as we also have go.mod and go.sum over there as well.

sure, will do.

Reply:
>I don't quite get it yet: Why should there be a change there?
Do you mean that the operations like go mod tidy could lead to a change there and that should result in test failure?

yes that's correct, that can result in a change, and file content will change, we have a similar check-in 

https://github.com/ceph/ceph-csi/blob/d1c28fa57a0319879dc516ade77c5ec0d90500e9/Makefile#L110-L114